src/test/java/com/alipay/oceanbase/hbase/OHTableTest.java

```diff
@@ -41,9 +43,38 @@ public void setup() throws IOException {
 
         Configuration c = ObHTableTestUtil.newConfiguration();
         c.set("rs.list.acquire.read.timeout", "10000");
+        c.set(Property.RPC_CONNECT_TIMEOUT.getKey(), "1000000000");
+        c.set(Property.RPC_OPERATION_TIMEOUT.getKey(), "1000000000");
+        c.set(Property.RPC_EXECUTE_TIMEOUT.getKey(), "10000000000");
         hTable = new OHTable(c, "test");
     }
 
+    @Test
+    public void testParseNamespace() throws IOException {
```

Comment:
add end-end test for namespace?

Reply:
When constructing ohTable, the database corresponding to the observer will be connected